README.md

```diff
@@ -68,15 +68,27 @@ update:
     labels: ["WIP", "Update Me"]
 ```
 
-### Caveats and Notes
-
-If both `blacklist` and `whitelist` are specified, bulldozer will attempt to match on both. 
-In cases where both match, `blacklist` will take precedence.
-
-The `merge_method` specifies the strategy that will be used to merge. Possible choices
-are `merge`, `squash`, and `rebase`. Specifying `squash` will allow for a further
-set of `squash_strategy` options, `pull_request_body`, `summarize_commits` and
-`empty_body` that will constitute the body of the merge commit message. 
+## FAQ
+
+#### Can I specify both a `blacklist` and `whitelist`?
+Yes. If both `blacklist` and `whitelist` are specified, bulldozer will attempt to match
+on both. In cases where both match, `blacklist` will take precedence.
+
+#### Can I specify the body of the commit when using the `squash` strategy?
+Yes. Specifying `squash` will allow for a further set of `squash_strategy` options,
```

Comment:
```
Yes. When the merge strategy is `squash`, you can set additional options under the `options.squash` property, including how to render the commit body.
```

This could also use a small configuration snippet, since the options are a nested structure.